test/fake_app/app/views/authors/show.html.erb

```diff
@@ -1,8 +1,8 @@
 <%= @author.name %>
 <%= @author.capitalized_name %>
 <%= @author.books.first.upcased_title %>
-<%= @author.books[-1].upcased_title %>
-<% if p = @author.publishers.last %><%= p.upcased_name %><% end %>
+<%= @author.books.last.upcased_title %>
+<% if Rails.version.to_f > 5.0 && p = @author.publishers.second_to_last %><%= p.upcased_name %><% end %>
```

Comment:
`#second_to_last` and `#third_to_last` were added in Rails 5.0.0. https://github.com/rails/rails/commit/074ff5ce147bdbc988b54dbc5fb60f6eae9506a9